### PR TITLE
Adding the first mass spectrometry imaging tutorial

### PR DESCRIPTION
**Pull request by foellmelanie**
I will make prettier pictures within the next weeks.

### CODE REVIEW

**Comment by shiltemann** [comment]
very cool @foellmelanie! I just made a few very small changes to the formatting (mainly using snippets where possible) but this looks great, thanks a lot!

File: topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/data-library.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (bebatut) [inline topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/data-library.yaml]: you could remove this file

File: topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tools.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (bebatut) [inline topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tools.yaml]: And also removing this one

**Comment by shiltemann** [comment]
(current travis failures are unrelated to this tutorial and will be fixed with https://github.com/galaxyproject/training-material/pull/1165)

**Comment by bgruening** [comment]
Thanks all!

**Comment by foellmelanie** [comment]
thanks a lot for the fast review @shiltemann @bebatut !

**Pull request merged**